CMakeLists.txt

```diff
@@ -297,12 +297,16 @@ if (TEST_INTEL_OPTIMIZATION)
 endif()
 
 add_subdirectory(src)
-add_subdirectory(example)
+add_subdirectory(include/parthenon)
```

Comment:
Why the additional subdirectory - why not just put the files directly under include?

Reply:
This follows my understanding of convention, but I can change it if needed. https://cliutils.gitlab.io/modern-cmake/chapters/basics/structure.html, https://hsf-training.github.io/hsf-training-cmake-webpage/06-projectstructure/index.html.

Reply:
Sorry, I didn't see this response - you're right, that all of the headers *should* go under a `parthenon` directory. In fact, that's the path they should be included from in the app. That's why when I added the explicitly public headers (e.g. `parthenon/prelude.hpp`, etc.), I put them in a `parthenon` directory.

Unfortunately, most of our includes don't currently work like that - they are expected to live at the "root" directory. Hopefully, we can over time move all of our header files into the `parthenon` directory over time.

My opinion is that the `include` directory should match the structure that the user is expected to include them, even if that structure is less than desirable.

Reply:
Ok, I see, that makes this a bit awkward.